core/src/test/java/org/apache/calcite/test/ReflectiveSchemaTest.java

```diff
@@ -585,10 +585,9 @@ private void check(ResultSetMetaData metaData, String columnName,
     with.query("select \"wrapperLong\" / \"primitiveLong\" as c\n"
         + " from \"s\".\"everyTypes\" where \"primitiveLong\" <> 0")
         .planContains(
-            "final Long inp13_ = current.wrapperLong;")
+            "final Long input_value = current.wrapperLong;")
         .planContains(
-            "return inp13_ == null ? (Long) null "
-                + ": Long.valueOf(inp13_.longValue() / current.primitiveLong);")
+            "return input_value == null ? (Long) null : Long.valueOf(input_value / Long.valueOf(current.primitiveLong));")
```

Comment:
Isn't this unnecessary boxing/unboxing?

Reply:
The difference is 
```
old: inp13_.longValue() / current.primitiveLong)
new: input_value / Long.valueOf(current.primitiveLong)
```
We can conduct unboxing optimization before operands entering into `BinaryImplementor`.
Thanks, let me change it.

Reply:
Hi @zabetak, comment addressed, unboxing optimization if necessary is conducted. Keep the same as that generated by current code generator.